sharkfuser/include/fusilli/support/asm_emitter.h

```diff
@@ -554,6 +554,256 @@ inline std::string ConvFPropNode::emitNodePreAsm() const {
   return output;
 }
 
+//===----------------------------------------------------------------------===//
+//
+// ConvWGradNode ASM Emitter Methods
+//
+//===----------------------------------------------------------------------===//
+
+// Emits ConvWGradNode's operand names in MLIR assembly format.
+//
+// Its output is used to materialize the contents of {} in
+//      %result = torch.aten.convolution_backward {}, ...
+// with
+//      "%arg0_dy, %arg1_x"
+inline std::string ConvWGradNode::getOperandNamesAsm() const {
+  return convWGradAttr.getDY()->getValueNameAsm() + "_permuted" + ", " +
+         convWGradAttr.getX()->getValueNameAsm() + "_permuted";
```

Comment:
minor nit: keep the suffix consistent between this and `ConvFPropNode` (either is fine - `_permuted` or `_perm`).